aiden-recommender/aiden_recommender/scrapers/abstract_parser.py

```diff
@@ -0,0 +1,15 @@
+from aiden_recommender.models import JobOffer
+from aiden_recommender.scrapers.field_extractors import AbstractFieldExtractor
+from abc import ABC
+
+
+class AbstractParser(ABC):
```

Comment:
I honestly don't like this design, the idea of listing the attributes with dir seems to be so dirty to me. Any attribute can mean anything, why not simply have a `add_field` function or something similar that given a field and an extractor it adds an entry to the data dict ?